gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -636,23 +689,22 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
 
    if (!CreateHttpServer(true)) {
       R__ERROR_HERE("WebDisplay") << "Fail to start real HTTP server";
-      return false;
+      return 0;
    }
 
    addr = fAddr + addr;
 
-   int port = gEnv->GetValue("WebGui.HeadlessPort", 9222);
-   exec.ReplaceAll("$port", std::to_string(port).c_str());
    exec.ReplaceAll("$url", addr.c_str());
    exec.ReplaceAll("$width", swidth.c_str());
    exec.ReplaceAll("$height", sheight.c_str());
 
    if (exec.Index("fork:") == 0) {
       exec.Remove(0, 5);
 #if !defined(_MSC_VER)
+
       std::unique_ptr<TObjArray> args(exec.Tokenize(" "));
       if (!args || (args->GetLast()<=0))
-         return false;
+         return 0;
```

Comment:
I think an empty `fork:` entry is a bug and warrants at least a warning.

Reply:
Yes